src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -14613,40 +16032,102 @@ void Assembler::shrq(Address dst, int imm8) {
   }
 }
 
+void Assembler::eshrq(Register dst, Address src, int imm8, bool no_flags) {
+  InstructionMark im(this);
+  assert(isShiftCount(imm8 >> 1), "illegal shift count");
+  InstructionAttr attributes(AVX_128bit, /* vex_w */ true, /* legacy_mode */ false, /* no_mask_reg */ true, /* uses_vl */ false);
+  attributes.set_address_attributes(/* tuple_type */ EVEX_NOSCALE, /* input_size_in_bits */ EVEX_32bit);
```

Comment:
input_size_in_bits should be EVEX_64bit.

Reply:
Thanks, done.